DataPlotly/layouts/plot_layout_item.py

```diff
@@ -46,7 +50,7 @@ def javaScriptConsoleMessage(self, message, lineNumber, source):
         QgsMessageLog.logMessage('{}:{} {}'.format(source, lineNumber, message), 'DataPlotly')
 
 
-class PlotLayoutItem(QgsLayoutItem):
+class PlotLayoutItem(QgsLayoutItem, QgsExpressionContextGenerator):
```

Comment:
QgsLayoutItem is already a QgsExpressionContextGenerator - no need for the extra inheritance here